content_scripts/CameraFollowCoordinator.js

```diff
@@ -130,7 +124,14 @@ export class CameraFollowCoordinator {
             console.warn('Can\'t find a virtualizer to follow');
             return;
         }
-        this.follow(followTarget.id);
+        realityEditor.envelopeManager.focusEnvelope(followTarget.followable.frameKey);
+        this.follow(followTarget.id, this.followDistance);
+    }
+    close2D_UI() {
```

Comment:
This should be `close2D` or `close2DUI`, the underscore isn't really used in any other method names so it would cause confusion